Tests/MeiliSearchIntegrationTests/TaskTests.swift

```diff
@@ -57,31 +57,36 @@ class TasksTests: XCTestCase {
   }
 
   func testGetTasksIndex() {
-    let addDocExpectation = XCTestExpectation(description: "Add documents")
-
-    addDocuments(client: self.client, uid: self.uid, primaryKey: nil) { result in
-      switch result {
-      case .success:
-        addDocExpectation.fulfill()
-      case .failure(let error):
-        dump(error)
-        XCTFail("Failed to create index")
-        addDocExpectation.fulfill()
-      }
-    }
-    self.wait(for: [addDocExpectation], timeout: TESTS_TIME_OUT)
+    let expectation = XCTestExpectation(description: "Add documents")
```

Comment:
not sure the description is still relevant here

Reply:
CTRL+C CTRL+V spotted!